integration-cli/requirements_unix.go

```diff
@@ -81,6 +81,18 @@ var (
 		},
 		"Test requires that seccomp support be enabled in the daemon.",
 	}
+	bridgeNfIptables = testRequirement{
+		func() bool {
+			return !SysInfo.BridgeNfCallIptablesDisabled
+		},
+		"Test requires that bridge-nf-call-iptables support be enabled in the daemon.",
+	}
+	bridgeNfIP6tables = testRequirement{
+		func() bool {
+			return !SysInfo.BridgeNfCallIptablesDisabled
```

Comment:
@coolljt0725 I think that should be `BridgeNfCallIp6tablesDisabled`, shouldn't it?